### PR TITLE
sync governance description

### LINKED ISSUE #98456: README governance is out of sync with the changes
### 📚 The doc issue

README.md and https://github.com/pytorch/pytorch/blob/master/docs/source/community/governance.rst

indicate different governance structure, different maintainers, etc..

### Suggest a potential alternative/fix

update the readme to point to the update governance documentation .. modify the current readme contents to be a thank you for Emeritus efforts section.

### PR DESCRIPTION
**Pull request by mikebrow**
Fixes #98456

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -435,7 +435,11 @@ To learn more about making a contribution to Pytorch, please see our [Contributi
 
 PyTorch is a community-driven project with several skillful engineers and researchers contributing to it.
 
-PyTorch is currently maintained by [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) with major contributions coming from hundreds of talented individuals in various forms and means.
+PyTorch has adopted a [hierarchical technical governance structure.](https://github.com/pytorch/pytorch/blob/master/docs/source/community/governance.rst)
+
+### Thank You
+
+PyTorch would like to recognize Emeritous maintainer efforts comming from [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) as well as major contributions coming from hundreds of talented individuals in various forms and means.
```
Comment (awgu) [inline README.md:442]: nit: just passing by to mention some typos
```suggestion
PyTorch would like to recognize Emeritus maintainer efforts coming from [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) as well as major contributions coming from hundreds of talented individuals in various forms and means.
```

Comment (mikebrow) [reply README.md]: thx addressed..

**Comment by gottbrath** [comment]
Mike,

Thanks for flagging that the readme mentions some emeritus folks. The PR https://github.com/pytorch/pytorch/pull/98457 proposed above to fix this seems to suggest that Greg and Soumith are emeritus which isn't correct.

If you want to revise the readme document I like the reference to the hierarchical governance document and I think recognizing the current core maintainers by name would be good. The current core maintainer list is

Soumith Chintala ([soumith](https://github.com/soumith))

Edward Yang ([ezyang](https://github.com/ezyang))

Greg Chanan ([gchanan](https://github.com/gchanan))

Dmytro Dzhulgakov ([dzhulgakov](https://github.com/dzhulgakov))

Nikita Shula ([malfet](https://github.com/malfet))

Alternately, if you want to I can absorb updating the readme.md into my WIP PR https://github.com/pytorch/pytorch/pull/98520

**Comment by mikebrow** [comment]
> Mike,
> 
> Thanks for flagging that the readme mentions some emeritus folks. The proposed fix however seems to suggest that Greg and Soumith are emeritus which isn't correct.
> 
> If you want to revise the readme document I like the reference to the hierarchical governance document and I think recognizing the current core maintainers by name would be good. The current core maintainer list is
> 
> Soumith Chintala ([soumith](https://github.com/soumith))
> 
> Edward Yang ([ezyang](https://github.com/ezyang))
> 
> Greg Chanan ([gchanan](https://github.com/gchanan))
> 
> Dmytro Dzhulgakov ([dzhulgakov](https://github.com/dzhulgakov))
> 
> Nikita Shulga ([malfet](https://github.com/malfet))
> 
> Alternately, if you want to I can absorb updating the readme.md into my WIP PR #98520

sure absorb away.. Cheers, Mike

**Pull request closed without merging**